nextflow.config

```diff
@@ -128,6 +128,12 @@ params {
   freebayes_C = 1
   freebayes_g = 0
   freebayes_p = 2
+  pileupcaller_snpfile = ''
```

Comment:
Can you add a //Comment here that these are all options for pileupCaller? Just to be clearer as to what belongs to freebayes / other tools. (purely cosmetic, code is fine of course!)